web-platform/website/src/components/navigation-bar/NavigationBar.tsx

```diff
@@ -70,11 +70,11 @@ function DropdownMenus(props: { user?: User }) {
 
   return (
     <>
-      <Col className="d-flex me-0 justify-content-end d-lg-none d-xl-none d-xxl-none">
+      <Col className="d-flex me-0 justify-content-end d-lg-none">
```

Comment:
I suggest grouping the `d-*` classes together

Reply:
Done